stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/ChatUI.kt

```diff
@@ -0,0 +1,44 @@
+package io.getstream.chat.android.ui
+
+import android.content.Context
+import com.getstream.sdk.chat.style.ChatFonts
+import com.getstream.sdk.chat.style.ChatStyle
+import io.getstream.chat.android.ui.common.UrlSigner
+import io.getstream.chat.android.ui.common.markdown.ChatMarkdown
+import io.getstream.chat.android.ui.common.markdown.ChatMarkdownImpl
+import io.getstream.chat.android.ui.common.style.ChatFontsImpl
+
+/**
+ * ChatUI handles any configuration for the Chat UI elements. It replaces the older Chat class.
+ *
+ * @param fonts allows you to overwrite fonts
+ * @param markdown interface to to customize the markdown parsing behaviour, useful if you want to use more markdown modules
+ * @param urlSigner url signing logic, enables you to add authorization tokens for images, video etc
+ *
+ * @see ChatMarkdown
+ * @see UrlSigner
+ * @see ChatFonts
+ */
+public object ChatUI {
```

Comment:
A new ui-component's `ChatUI` implementation is initialized automatically. No need for the user to do anything.